### DISCUSSION TITLE
💡 Proposing to club generic wrappid templates with toolkit

### DISCUSSION THREAD

**Discussion by anantakumarghosh** [wrappid/.github#6] [Ideas]
### Hi @wrappid/core-team  🙌 
I have an interesting idea
## What if we club generic wrappid templates(app, module, service) with toolkit?
(**generic** term refers to initial wrappid templates: wrappid-app, wrappid-module, wrappid-service)

### Current Behavior:
Toolkit copies wrappid template into org/user's repository

### Proposing:
- Wrappid templates will be inside toolkit package, templates will be cloned/pulled before build-publish process

- If there are some more template variation(not generic) or custom app template, like a wrappid-app that looks completely different,
  for example let us assume it as a template named: wrappid-lightApp

- Toolkit won't have it. Instead users will have to download and import custom template in toolkit with some command like, 
  `wrappid template add [path of template]`.

Feel free to share your thoughts, pros and cons 😄

**Comment by samhere17** [comment]
I am not sure how that will help

**Reply by techoneel** [reply]
@anantakumarghosh do you have anything in mind cause I agreed with @samhere17 don''t know how it can be helpful.

**Reply by anantakumarghosh** [reply]
> @anantakumarghosh do you have anything in mind cause I agreed with @samhere17 don''t know how it can be helpful.

No.

**Comment by anantakumarghosh** [comment]
Closing this discussion as not valid on today's date.
